runpod_worker/worker.py

```
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+"""
+CharForgex RunPod Serverless Worker
+Seamless serverless worker following RunPod patterns
+"""
+
+import runpod
+import os
+import sys
+import logging
+import time
+from typing import Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)